boards/nrf52840dongle/reset.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C)  2020 Benjamin Valentin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nrf52840dongle
+ * @{
+ * @file
+ * @brief       Implementation for managing the nrfutil bootloader
```

Comment:
I prefer a more practical description something that contains like "allows reboot into bootloader via hardware reset".
At least you could specify "managing" somehow.
This is nonblocking though.

Reply:
Sure! May I squash directly?

Reply:
yeah, please squash right away.